### PR TITLE
inception

### PR DESCRIPTION
**Pull request by pandeysk**
What to write here

Add address to #45

### CODE REVIEW

File: bios/pandeysk-bio.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+## a little about me
+ - I wish my life had versioning
+ - I could do multiple things on differnet branches
```
Comment (abhijithrr) [inline bios/pandeysk-bio.md]: Oh! Wow.. Very Good..

Comment (ManishVerma2805) [inline bios/pandeysk-bio.md]: Hey u there, why u need versioning on life.

BOSS life has no retakes :)

File: bios/pandeysk-bio.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+## a little about me
+ - I wish my life had versioning
```
Comment (ManishVerma2805) [inline bios/pandeysk-bio.md:2]: life starts with v1.0 and ends with v1.0

**Comment by pandeysk** [comment]
Looks like... now that I am encouraged with your comment, let me add something more.

File: bios/pandeysk-bio.md (hunk 2 above)
Comment (abhijithrr) [inline bios/pandeysk-bio.md:2]: @ManishVerma2805 Wow!! What a gyan… :+1:

File: bios/pandeysk-bio.md (hunk 1 above)
Comment (pandeysk) [inline bios/pandeysk-bio.md]: That's why it is just a wish list ;)

**Pull request merged**